data/xml/2024.propor.xml

```diff
@@ -756,4 +756,333 @@
       <bibkey>freitas-2024-text</bibkey>
     </paper>
   </volume>
+  <volume id="2" ingest-date="2024-05-01" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of the 16th International Conference on Computational Processing of Portuguese</booktitle>
+      <editor><first>Pablo</first><last>Gamallo</last></editor>
```

Comment:
We need a vol 1 / vol 2 in the book title — look at the PDF to see how to format it (add to both volumes)

Reply:
poke @anthology-assist

Reply:
Done.